test/spec.index.js

```diff
@@ -850,6 +850,34 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('looks up field label from fields.field-name.options.foo.label if not specified', function () {
+                middleware = mixins({
+                    'field-name': {
+                        options: ['foo', 'bar']
+                    }
+                });
+                middleware(req, res, next);
+                res.locals['radio-group']().call(res.locals, 'field-name');
+                render.args[0][0].options[0].label.should.be.equal('fields.field-name.options.foo.label');
+                render.args[0][0].options[1].label.should.be.equal('fields.field-name.options.bar.label');
+            });
+
+            it('looks up field label from fields.field-name.options.foo.label if not specified (object options)', function () {
```

Comment:
Field names in tests don't make sense. Would something like 'looks up field label from translations when the option is defined as an object' be better?